fixtures/martin/quantum-wash.json

```diff
@@ -0,0 +1,177 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Quantum Wash",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Ryan Brady"],
+    "createDate": "2023-12-19",
+    "lastModifyDate": "2023-12-19"
+  },
+  "links": {
+    "manual": [
+      "https://www.martin.com/en-US/products/mac-quantum-wash"
+    ]
+  },
+  "rdm": {
+    "modelId": 34654457,
+    "softwareVersion": "V1"
+  },
+  "availableChannels": {
+    "Shutter / Strobe": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ShutterStrobe",
+        "shutterEffect": "Open"
```

Comment:
Needs to break out: closed, open, strobe (range), open, random (range).

Default is actually 30 (middle of the first Open range).